spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkCopyOnWriteScan.java

```diff
@@ -123,8 +127,17 @@ public void filter(Filter[] filters) {
               tasks().stream()
                   .filter(file -> fileLocations.contains(file.file().path().toString()))
                   .collect(Collectors.toList());
+
+          LOG.info(
+              "{}/{} tasks for table {} matched runtime file filter",
+              filteredTasks.size(),
+              tasks().size(),
+              table().name());
+
           resetTasks(filteredTasks);
         }
+      } else {
+        LOG.warn("Unsupported runtime filter {}", filter);
       }
```

Comment:
Should this be a warn? Previously this seemed like an acceptable case , just wondering if it'll confuse users who look at the logs.

Reply:
I overlooked this in the initial implementation. There should be no use case where we can't convert a runtime filter for the copy-on-write scan. If it happens, we should warn the user.

Reply:
Got it thanks for the explanation